BrainCloudClient/tests/TestScript.cs

```diff
@@ -41,6 +41,20 @@ public void TestScheduleScriptUTC()
             tr.Run();
         }
 
+        [Test]
+        public void TestScheduleRunScriptUTCv2()
+        {
+            TestResult tr = new TestResult(_bc);
+
+            _bc.ScriptService.ScheduleRunScriptUTCv2(
+                _scriptName,
+                Helpers.CreateJsonPair("testParm1", 1),
+                (UInt64)((TimeZoneInfo.ConvertTimeToUtc(DateTime.UtcNow) - new DateTime(1970, 1, 1, 0, 0, 0, 0, DateTimeKind.Utc)).TotalMilliseconds),
```

Comment:
does this work? is this the official way of doing it? We will have to update the log on the client API on the portal. I'm gonna bring @stevemrjones to make sure this is proper way of doing it. Then you can change the test from his feedbacks and see if it still works.

Reply:
I would use a UTIL method to help this conversion